### DISCUSSION TITLE
Spotify Codecs Still Unsupported even with proprietary_codecs=true ffmpeg_branding=Chrome

### DISCUSSION THREAD

**Discussion by WolfJMZ** [cefsharp/CefSharp#4253] [General]
Hi there,

I'm trying to make a web player for spotify that uses the spotify web sdk - my html page is fine since it works without issues in chrome and edge.

I tried using my html page to see if it'll play inside of cefsharp but the result I get in the dev tools console is this:
[0919/143916.787:INFO:CONSOLE(18)] "Uncaught NotSupportedError: Failed to execute 'addSourceBuffer' on 'MediaSource': The type provided ('audio/mp4; codecs="mp4a.40.2"') is unsupported." (18)

I am building with proprietary codecs since youtube videos play just fine within the browser.

Can someone help me figure out how to get this enabled?

**Comment by amaitland** [comment]
Best place to ask for help building CEF is https://magpcss.org/ceforum/index.php

If you built cefclient I'd also suggest testing with that first to confirm your build is correct. 

I personally haven't built a version with proprietary codecs enabled for 6+ months, maybe something changed.

**Comment by WolfJMZ** [comment]
I actually got the build to work - but now there's an issue with widevine - the spotify client is returning a 403 when trying to open the spotify website in the cefclient.exe sample app - 

 https://gue1-spclient.spotify.com/widevine-license/v1/audio/license 403

Ugh. Yep to the forums I go. This is so dumb and painful.

**Reply by amaitland** [reply]
Asked at https://magpcss.org/ceforum/viewtopic.php?f=6&t=19200

Looks like additional `Widevine License` is required.

**Reply by WolfJMZ** [reply]
Yes I'm the one that opened that topic - additionally I've sent multiple requests to widevine and heard nothing.

**Reply by amaitland** [reply]
> Yes I'm the one that opened that topic

I know (you used the same username) 👍

Posting as reference for anyone else that ends up here looking for information.

> additionally I've sent multiple requests to widevine and heard nothing.

Personally I'd expect a response to take weeks not days.
